go.sum

```diff
@@ -1,4 +1,6 @@
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+github.com/FrangipaneTeam/terraform-plugin-framework-validators v0.0.0-20230301104912-dc064f6846d6 h1:FZaUvldOocJ+aCMqZuoFOQHWknN12c+ufjdaSbN69BQ=
+github.com/FrangipaneTeam/terraform-plugin-framework-validators v0.0.0-20230301104912-dc064f6846d6/go.mod h1:adSDKHqEGbCnCthXGrQl8UurJzX/RO9bod8OeMJ58o8=
```

Comment:
Make a PR for specific enhancement.

Reply:
It's not an enhancement is for use the validator in resource.